openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateCoilHeatingGas.cpp

```diff
@@ -73,7 +73,12 @@ boost::optional<IdfObject> ForwardTranslator::translateCoilHeatingGas( CoilHeati
   ///////////////////////////////////////////////////////////////////////////
   // Field: Fuel Type
   ///////////////////////////////////////////////////////////////////////////
-  idfObject.setString(openstudio::Coil_Heating_FuelFields::FuelType, modelObject.fuelType());
+  if (modelObject.fuelType() == "PropaneGas") {
+    idfObject.setString(openstudio::Coil_Heating_FuelFields::FuelType, "Propane");
```

Comment:
You can add a LOG(Warn, "'PropaneGas' is deprecated for Coil_Heating_FuelFields:FuelType, use 'Propane' instead")